lib/tokens.js

```diff
@@ -449,6 +465,41 @@ export async function resolveToInternalId({token}) {
   return {internalId};
 }
 
+/**
+ * Updates a token batch based off of `internalId`.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.internalId - The internal ID of the token batch.
+ * @param {number} options.minAssuranceForResolution - Minimum level of
+ *   assurance required for token resolution.
+ *
+ * @returns {Promise<boolean>} Returns true if update occurred.
+ */
+export async function updateBatches({internalId, minAssuranceForResolution}) {
```

Comment:
I recommend we rename this to `setMinAssuranceForResolution` to keep it specific to its job. Depending on our security analysis elsewhere, we may find we need `raiseMinAssuranceForResolution` and `lowerMinAssuranceForResolution` (or similar) functions.